scripts/migrate-auth0-users.js

```diff
@@ -0,0 +1,137 @@
+const fetch = require('isomorphic-fetch')
+const omit = require('lodash/omit')
+const uuid = require('uuid/v4')
+
+// the below json files should have the form of:
+// {
+//   apiClient: 'SECRET',
+//   apiSecret: 'SECRET',
+//   domain: 'SECRET.auth0.com',
+//   clientId: 'SECRET'
+// }
+const from = require('./from-client.json')
+const to = require('./to-client.json')
```

Comment:
Should we add `.json.tmp` files for these? Also, it might be worth adding them to `.gitignore`.

Reply:
I think it's enough to include the example in the comments. I sent you the secrets privately.